### ISSUE TITLE
[Bug]: Raya Lucaria academy crash after entering gate

### ISSUE THREAD

**Issue by loko-99jkjk** [LukeYui/EldenRingSeamlessCoopRelease#289]
### Describe the issue in detail

4 of us were playing and when we got to Raya Lucaria academy gate , the host telported in and his game crashed and he was not able to start the game after that because it kept crashing as soon as he loads in , this happned again when one of us was abducted by the virgin abductor andn he wasnt able to log in and the game kept crashing as soon as he logs in.

### Steps to recreate the bug

Go to Raya Lucaria academy and use the key 
Get abduced by virgin abductor

### Version

1.2.5 (Latest)

**Comment by jrhooper7** [comment]
I'm having the same issue. me and 2 other friends, 1 hit the gate and they loaded in while I've been stuck in loading screen limbo forever.

**Comment by zaylivinglive** [comment]
Hey there, we had this issue as well. This happened to me when the non host went into the academy gate first, and not the host. It broke the game and he could not launch without crashing. We fixed his game simply by verifying game integrity on steam, launching game, fast traveling out of the academy offline, then rejoining the game. After we went back to the academy, having me (the host) go in the gate first allowed us to complete the entire academy with no issues. Hope this helps.
